### PR TITLE
[webapi] Notification automation

### PR DESCRIPTION
**Pull request by kangxu**
Impacted tests(approved): new 0, update 2, delete 0
Unit test platform: Crosswalk for Android 15.44.384.0
Unit test result summary: pass 2, fail 0, block 0

### CODE REVIEW

**Comment by Honry** [comment]
Please split this PR into two, since you both update the ATIP and test cases.

**Comment by kangxu** [comment]
@Honry I have split this PR into two PRs following your comments, and update ATIP in https://github.com/crosswalk-project/crosswalk-test-suite/pull/2705

**Comment by kangxu** [comment]
@Honry  I have updated steps in this PR sync with https://github.com/crosswalk-project/crosswalk-test-suite/pull/2705

File: webapi/tct-notification-w3c-tests/suite.json [hunk 1]
```diff
@@ -16,14 +16,30 @@
                 "*"
             ],
             "copylist": {
+                "PACK-TOOL-ROOT/resources/bdd/bddrunner": "bddrunner",
+                "PACK-TOOL-ROOT/resources/bdd/data.conf": "data.conf",
```
Comment (Honry) [inline webapi/tct-notification-w3c-tests/suite.json]: data.conf is only required in baseline tests.

**Comment by kangxu** [comment]
@Honry  I have removed "data.conf" section in suite.json

**Comment by zqzhang** [comment]
> Please split this PR into two, since you both update the ATIP and test cases.

Actually, you can make 2 commits in a pull request for 2 purposes if they have close relationship.

**Comment by Honry** [comment]
@zqzhang, thank you for your comment, new tips got. :smile:

**Pull request merged**